R/learner_aorsf_surv_aorsf.R

```diff
@@ -0,0 +1,182 @@
+#' @title Accelerated Oblique Random Survival Forest Learner
+#' @name mlr_learners_surv.aorsf
+#' @author bcjaeger
+#'
+#' @description
+#' Accelerated oblique random survival forest.
+#' Calls [aorsf::orsf()] from \CRANpkg{aorsf}.
+#' @template learner
+#' @templateVar id surv.aorsf
+#'
+#' @section Custom mlr3 parameters:
+#' - `mtry`:
+#'   - This hyperparameter can alternatively be set via the added hyperparameter `mtry_ratio`
+#'     as `mtry = max(ceiling(mtry_ratio * n_features), 1)`.
+#'     Note that `mtry` and `mtry_ratio` are mutually exclusive.
+#'
+#' @references
+#' `r format_bib("jaeger_2019")`
+#'
+#' `r format_bib("jaeger_2022")`
+#'
+#'
+#' @template seealso_learner
+#' @template example
+#' @export
+delayedAssign(
+  "LearnerSurvAorsf",
+  R6Class("LearnerSurvAorsf",
+          inherit = mlr3proba::LearnerSurv,
+          public = list(
+            #' @description
+            #' Creates a new instance of this [R6][R6::R6Class] class.
+            initialize = function() {
+              ps = ps(
+                n_tree = p_int(default = 500L, lower = 1L, tags = 'train'),
+                n_split = p_int(default = 5L, lower = 1L, tags = 'train'),
+                n_retry = p_int(default = 3L, lower = 0L, tags = 'train'),
+                mtry = p_int(default = NULL, lower = 1L,
+                             special_vals = list(NULL), tags = 'train'),
+                mtry_ratio = p_dbl(lower = 0, upper = 1, tags = "train"),
+                control_type = p_fct(levels = c("fast", "cph", "net"),
+                                     default = "fast", tags = "train"),
+                control_fast_do_scale = p_lgl(default = TRUE, tags = "train"),
+                control_fast_method = p_fct(levels = c("efron", "breslow"),
```

Comment:
here the default is set to "breslow", but when I look at the docs from orsf_control_fast, the default is "efron". 
Please document this in the learner documentation under the section "custom mlr3 defaults" like here: https://github.com/mlr-org/mlr3extralearners/blob/56d38b1401bad469631aa912101d659caa3ad536/R/learner_lightgbm_regr_lightgbm.R#L26

Please also explain these custom parameters that are not actually documented in the aorsf package (like control_fast_do_scale). Otherwise one has to look at the code, or guess what they do

Reply:
Please also check this for the other custom parameters (i.e. parameters whose name is not identical to the name of a parameter in the functions from the aorsf package)

Reply:
My mistake - "efron" was the method I intended to put as the default here. After reviewing other inputs for `orsf` and the control functions, I found no other discrepancies between default values.